gradoop-data-integration/src/main/java/org/gradoop/dataintegration/transformation/functions/NeighborhoodVertex.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.dataintegration.transformation.functions;
+
+import org.gradoop.common.model.impl.id.GradoopId;
+
+/**
+ * A simple neighbor vertex pojo which contains information about the Id, the connecting edge id
+ * and label.
+ */
+public class NeighborhoodVertex {
```

Comment:
Maybe `extends Tuple2<GradoopId, String>` to promote the optimizer process of flink.

Reply:
You're right, will do that